docs/experimentalists/pooler/random/index.md

```diff
@@ -22,8 +22,9 @@ This means that there are 9 possible combinations for these variables (3x3), fro
 | 3  | (3,4) | (3,5) | X   |
 
 ### Example Code
+
 ```python
-from autora.experimentalist.pooler.random_pooler import random_pool
+from autora.experimentalist.random_ import random_pool
```

Comment:
Same here. What is the reasoning for using `_` in this case, and not in other cases?

Reply:
`random` is a different case – that's the name of a python built-in module (as in `from random import seed, choice`). We shouldn't have a thing with the same name as a built-in (qualified or unqualified) because that will lead to a lot of confusion, so we need a different name for that.

Reply:
`chance`? `stochastic`? 

the latter is probably already too loaded as a term with a particular technical meaning...?

Reply:
I've updated the GitHub Actions test runner to use `--import-mode importlib` so that we don't have this error any more. The module name is now `autora.experimentalist.random`